### DISCUSSION TITLE
I can't use thunder-client anymore after used function "View: Move Secondary Side Var Views To Panel"

### DISCUSSION THREAD

**Discussion by cartini-dominic** [thunderclient/thunder-client-support#1257] [Q&A] [answered]
<img width="466" alt="image" src="https://github.com/rangav/thunder-client-support/assets/119281759/44d1bd58-6a23-44b6-8db3-41276d4e717b">

Hello. Master.
I've often used this extension, it's very useful for me then Postman that need to open new window.
but someday, I can't use anymore. I don't know why it is showing this error message.
although, I tried uninstall and reboot and reinstall but I didn't have anything.
by any chance do you know why?
I just clicked "View: Move Secondary Side Var Views To Panel" in VSCode and my situation.
Thx you.

**Comment by rangav** [comment]
Hi @cartini-dominic 

thanks for reporting the problem.
What version of extension and vscode are you using?

**Reply by cartini-dominic** [reply]
Hello. I have
VSCode : Version: 1.78.2 (Universal)
Thunder Client: v2.10.2
Thank you!

**Reply by rangav** [reply, marked as answer]
Can you update vscode to latest version and try again?

**Reply by cartini-dominic** [reply]
wow. finally I can use all of them! thank you!!!
